javascript/react/intro-react/what-is-jsx.md

```diff
@@ -22,41 +22,59 @@ parent: what-is-react
 
 ---
 
-# What is **JSX**?
+# What is JSX?
 
 ---
 ## Content
 
-**JSX** is syntactic sugar that brings an `XML`-like syntax to **JavaScript**.
+**JSX** is syntactic sugar that brings an `XML`-like syntax to **JavaScript**. Here is how it looks like:
```

Comment:
I would say "HTML-like" syntax since HTML is more familiar to today's web devs than XML (and most people learning React will be web devs).